packages/graphback-schema/src/visitor.ts

```diff
@@ -1,7 +1,31 @@
-import { ObjectTypeDefinitionNode } from 'graphql';
+import { FieldDefinitionNode, ListTypeNode, NamedTypeNode, NameNode, NonNullTypeNode, ObjectTypeDefinitionNode } from 'graphql';
 
 export const visitor = {
```

Comment:
So visitor is going to collect whatever data that is needed and map that to digestible object. This is IMHO much cleaner approach than before where you were getting all.
Drawback here is that for each plugin we will need to visit entire graphql node couple times so it will affect overal performance. It is up to discussion/spike if we need couple dedicated plugins or large one. WDYT about this? Let's have bigger conversation on what we have in this pR already as we slowly heading towards something that will work for our use cases.